src/components/usa-add-aspect/usa-add-aspect.stories.js

```diff
@@ -0,0 +1,10 @@
+import "../../stylesheets/uswds.scss";
```

Comment:
Is the uswds.scss import necessary?

Reply:
Diving into all of these now! Doesn't look like importing the SCSS is necessary for this one!